go-controller/pkg/ovn/common.go

```diff
@@ -72,7 +72,7 @@ func (oc *Controller) setAddressSet(hashName string, addresses []string) {
 		return
 	}
 
-	ips := strings.Join(addresses, " ")
+	ips := "\"" + strings.Join(addresses, "\" \"") + "\""
```

Comment:
@russellb instead of all the escaping you can use backticks as the string start/end markers, eg \`"\` + blah + \`"\`

Reply:
:+1:

Reply:
done!